src/objc.h

```diff
@@ -0,0 +1,138 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (c) 2015 by Digital Mars
+ * All Rights Reserved
+ * written by Michel Fortin
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/D-Programming-Language/dmd/blob/master/src/objc.h
+ */
+
+#ifndef DMD_OBJC_H
+#define DMD_OBJC_H
+
+#include "root.h"
+#include "mtype.h"
+#include "stringtable.h"
+
+#include "objc_glue.h"
+
+class Identifier;
+class FuncDeclaration;
+class ClassDeclaration;
+class InterfaceDeclaration;
+class ObjcSelector;
+
+struct elem;
```

Comment:
Ok, I'll see. The only method that uses this is `ObjcSelector.toElem`. Otherwise there are only free functions, declared at the bottom, which are only called from the glue layer.